bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -103,7 +103,7 @@ export class Plot extends LayoutDOM
       title = if isString(@title) then new Title({text: @title}) else @title
       @add_layout(title, @title_location)
 
-    @_plot_canvas = new PlotCanvas({plot: @})
+    @_init_plot_canvas()
```

Comment:
I think `@_plot_canvas = @_init_plot_canvas()` (i.e. having the method return the canvas)  might be nicer than relying on the side effect of setting the property.